### PR TITLE
WMS-Phase3 | Some fixes

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?
* Does not allow checkbox event to be supressed
* Wrap dialog's title (to match the dialog's body width)

#### :man_shrugging: Why?
Changes required for WMS' Phase 3

#### :pushpin: Jira Issue
Not tracked.

#### :no_good: Blocked by
Not blocked, based on master.

#### :clipboard: Pending
Nothing

### CODE REVIEW

**Review by miguelcoba — APPROVED** [review]

File: src/UI/Checkbox.elm [hunk 1]
```diff
@@ -97,7 +96,7 @@ renderElement renderConfig (Checkbox { message, label, state } { labelVisible })
             , Border.color Palette.primary.middle
             , Border.width 2
             , Border.rounded 8
-            , Events.onClick (message (not state))
+            , Element.onIndividualClick (message (not state))
```
Comment (renanpvaz) [inline src/UI/Checkbox.elm:99]: wtf, where's this function coming from? I can't seem to find it in the docs

File: src/UI/Internal/Dialog.elm [hunk 2]
```diff
@@ -117,7 +114,7 @@ titleText cfg padding title =
     Text.heading5 title
         |> Text.renderElement cfg
         |> Element.el
-            [ Element.width fill
+            [ Element.width (px 400)
```
Comment (renanpvaz) [inline src/UI/Internal/Dialog.elm:117]: again if this is meant for mobile, might not fit on iphones

Comment (PedroHLC) [reply src/UI/Internal/Dialog.elm:117]: this wasn't supposed to be committed, oops

File: src/UI/Checkbox.elm (hunk 1 above)
Comment (PedroHLC) [reply src/UI/Checkbox.elm:99]: It's from our Internal.Element

**Review by segmentationfaulter — APPROVED** [review]

**Pull request merged**